src/test/java/api/InstancesApiExamples.java

```diff
@@ -54,17 +58,20 @@
 import api.support.ApiTests;
 import api.support.InstanceApiClient;
 import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.http.HttpMethod;
 import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
+import support.fakes.EndpointFailureDescriptor;
 
 public class InstancesApiExamples extends ApiTests {
 
   private static final InventoryConfiguration config = new InventoryConfigurationImpl();
   private final String tagNameOne = "important";
   private final String tagNameTwo = "very important";
 
-  public InstancesApiExamples() throws MalformedURLException {
-    super();
+  @After
+  public void expireFailureEmulation() throws Exception {
+    instancesStorageClient.expireFailureEmulation();
   }
```

Comment:
What does expire failure emulation mean?

Reply:
This means to disable failure emulation for instance-storage fake endpoint by setting `failureExpireDate` (when the endpoint should stop emulating failures) to a datetime in the past.

Reply:
Could it be renamed to `disableFailureEmulation` instead, given that is the intent?

```suggestion
  public void disableFailureEmulation() throws Exception {
    instancesStorageClient.expireFailureEmulation();
  }
```